src/ResourceManager/Compute/Commands.Compute.Test/Common/ComputeTestController.cs

```diff
@@ -26,9 +25,7 @@
 using Microsoft.Azure.Management.Network;
 using Microsoft.Azure.Management.KeyVault;
 #if NETSTANDARD
```

Comment:
Since the #if has no content now, you should just remove the #else and do `#if !NETSTANDARD`